release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1229,13 +1276,23 @@ In the table below, features are marked with the following statuses:
 |CSI inline ephemeral volumes
 |TP
 |TP
-|
+|TP
 
 |CSI vSphere Driver Operator
 |TP
 |TP
 |
 
+|OpenShift Shared Resources
+|-
+|-
+|TP
+
+|Shared Resource driver for CSI
+|-
+|-
+|TP
+
```

Comment:
These are the same thing, no?

I would also call it the "Shared Resource CSI Driver", to align with other CSI Driver names (example - "AWS Elastic Block Store CSI Driver")

Reply:
Updated. Thanks.